google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStandardDeployTask.java

```diff
@@ -47,6 +49,9 @@ public AppEngineStandardDeployTask(
 
   @Override
   public void execute(ProcessStartListener startListener) {
+    UsageTrackerProvider.getInstance()
```

Comment:
Done.

Determining if this is compat wasn't readily possible (due to required context missing), so I ended up trickling down that info from callers after changing function signatures.

One alternative is to ping events at the time of creating a Deploy Runner instead of doing it inside `execute()`. This also sounds a fine solution.